### ISSUE TITLE
Tags: Saving a post on a WoA site can result in an intermittent error.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#51640]
We recently fixed a bug in which [WP.com](<http://WP.com>)'s global handling of terms was conflicting with core's "split terms".

However, it appears this may need additional handling for Atomic, as we've a couple of reports of the same error popping up for Atomic since the fix went out on [WP.com](<http://WP.com>). It's also possible that the cause is unrelated, since I'm not sure if or how [WP.com](<http://WP.com>) global terms are implemented on the Atomic platform.

See: [https://github.com/Automattic/wp-calypso/issues/36274](<https://github.com/Automattic/wp-calypso/issues/36274>)  <br>Global vs. split terms background: p7H4VZ-2Vd-p2

**Comment by kwight** [comment]
[Reported](https://github.com/Automattic/wp-calypso/issues/36274#issuecomment-806389360) by @druesome :

> I just got another report of this happening in 28628867-hc. User says they entered a bunch of tags and wasn't able to save their draft. They said they added 17 or so tags. Reducing the number allowed them to save.

[Reported](https://github.com/Automattic/wp-calypso/issues/36274#issuecomment-810545318) by @katiebethbrown :

> I've got an atomic site in chat seeing the same error when adding a tag to posts in 27939677-hc. Happens with just a single tag, and I can replicate in a support session as well.

<img width="612" alt="terms" src="https://user-images.githubusercontent.com/42279181/113050760-70c95f80-9162-11eb-8291-c69d1339f583.png">

<img width="1002" alt="Screen Shot 2021-03-30 at 2 22 59 PM" src="https://user-images.githubusercontent.com/42279181/113051818-a3c02300-9163-11eb-8212-4c60317bf772.png">

**Comment by syds87** [comment]
Similar case seems to be reported in 29842652-hc. This just happens with a few categories, when trying to save as draft.

**Comment by vinnykaur** [comment]
The same user @syds87 had, appeared on chat again. 30045653-hc
They got really anxious that it's taking too long to get this resolved. 
Meanwhile is there a workaround that we can suggest to users?

**Comment by syds87** [comment]
Similar case reported on 4055322-zd

**Comment by inaikem** [comment]
Hi @kwight 👋 The same person mentioned above in 29842652-hc and 30045653-hc came back in to chat today asking for an update in 30690583-hc. I moved them over to 4156707-zd and resent the prev HE's suggested workaround.

Can you tell me if there's been any movement on issuing a fix for this since the last note? Thanks!

**Comment by edwinho89** [comment]
4156375-zd-woothemes

**Comment by kwight** [comment]
@inaikem No, I'm afraid there's no update and no expected date for a fix.

**Comment by goblinartificer** [comment]
30877970-hc ⁠— also Atomic, also happening even with just one category. Replicated in Firefox in a support session.

**Comment by mikeicode** [comment]
AT site - Floral theme, had that error and then the post published. 30875697-hc

**Comment by the-misha** [comment]
32143619-hc
AT site, Aster theme, errored but pubished.

**Comment by jordesign** [comment]
#35740706-hc
User returning to let us know issue is still happening for them.

**Comment by kmrpankaj** [comment]
#5438952-ZD
User came back to ask for an ETA.

**Comment by kspilarski** [comment]
Same case in 5462893-ZD. When adding more than 2 categories on a post, "Updating failed. Sorry, you are not allowed to assign the provided terms." message appears.

**Comment by jamiepalatnik** [comment]
Another report: 36525258-hc
